diff --git a/openmp/libomptarget/src/CMakeLists.txt b/openmp/libomptarget/src/CMakeLists.txt
index a40bb4dffd35..d5f56bfb42a8 100644
--- a/openmp/libomptarget/src/CMakeLists.txt
+++ b/openmp/libomptarget/src/CMakeLists.txt
@@ -63,8 +63,7 @@ target_include_directories(omptarget PRIVATE ${LIBOMPTARGET_INCLUDE_DIR})
 target_link_libraries(omptarget PRIVATE
   ${CMAKE_DL_LIBS}
   ${LLVM_OFFLOAD_ARCH}
-  "-Wl,--no-allow-shlib-undefined"
-  "-Wl,--version-script=${CMAKE_CURRENT_SOURCE_DIR}/exports")
+  "-Wl,--no-allow-shlib-undefined")
 
 # libomptarget.so needs to be aware of where the plugins live as they
 # are now separated in the build directory.
